### PR TITLE
Chore update logo

### PR DESCRIPTION
**Pull request by wismer**
New Logo! 
Screen Shot 2020-04-09 at 1 09 11 PM

### CODE REVIEW

File: docs/assets/postlight.svg [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
-</svg>
+<svg id="Layer_1" xmlns="http://www.w3.org/2000/svg" data-name="Layer 1" height="20" width="100" viewBox="0 0 73 13" >
+  <defs>
+    <style>
+    </style>
+    <clipPath id="clip-path" transform="translate(0 -0.66)">
+      <rect fill="none" y="0.66" width="72.75" height="12.65"/>
+    </clipPath>
+  </defs>
+  <title>postlight-logo-labs-13-2</title>
```
Comment (johnholdun) [inline docs/assets/postlight.svg:9]: Can we minify this SVG? I like using [SVGOMG](https://jakearchibald.github.io/svgomg/) for this, personally

File: docs/style.css [hunk 2]
```diff
@@ -273,7 +273,7 @@ footer p + p:before {
 .postlight-link {
   overflow: hidden;
   width: 6.3888888889rem; /* 115 / 27 */
```
Comment (johnholdun) [inline docs/style.css:275]: This value can probably change too. It looks like the new asset's 100 units wide by 20 units tall, so the width could be 1.3 * 100 / 20 = 6.5rem

File: docs/assets/postlight.svg (hunk 1 above)
Comment (wismer) [reply docs/assets/postlight.svg]: oh yea! good idea

**Comment by johnholdun** [comment]
Wonderful, thank you!

**Pull request merged**